### PR TITLE
refactor to React v16+

### PR DESCRIPTION
**Pull request by tyxou**
the Readme.md should be ignored from commit

### CODE REVIEW

File: src/Container.js [hunk 1]
```diff
@@ -70,17 +55,16 @@ export default class DialogContainer extends React.PureComponent {
...
-        backdropOpacity={0.3}
+      <Modal
+        transparent={true}
         style={styles.modal}
-        isVisible={visible}
-        animationIn={Platform.OS === "ios" ? IOS_MODAL_ANIMATION : "zoomIn"}
-        animationOut={"fadeOut"}
-        {...otherProps}
+        visible={visible}
+        animationType={(props.animationType)?props.animationType:"fade"}
```
Comment (mmazzarolo) [inline src/Container.js:62]: 🤔

**Review by mmazzarolo — CHANGES_REQUESTED** [review]
I'm experiencing a few issues running this version directly on iOS14 so I can't merge it as it is. I'll play around with it and update it where needed. Thanks!

**Comment by tyxou** [comment]
Haven't tested it with iOS 14, sorry about that.

**Comment by mmazzarolo** [comment]
@tyxou I integrated most of your changes in the new release: https://github.com/mmazzarolo/react-native-dialog/releases/tag/v6.0.0
Sorry if I haven't used your own branch but as you can see I also had a chance to work on other stuff in the same release :)
Thanks so much for your work!

**Pull request closed without merging**